scorm-tool/src/java/org/sakaiproject/scorm/ui/console/pages/res/scorm_console.css

```diff
@@ -193,3 +193,10 @@
     background-position: right center;
     padding-right: 16px;
 }
+
+.form-required label::before {
```

Comment:
Still need these in the scorm CSS?

Reply:
Yeah, we need this to make the form look a little more consistent with the rest of Sakai. I lifted this directly from Assignments. Before this, the required stars were black, and appeared after the label text (rather than being red and appearing before the label text).